### DISCUSSION TITLE
User Flows with custom-audit

### DISCUSSION THREAD

**Discussion by iamsethi** [GoogleChrome/lighthouse#15756] [Q&A] [answered]
I've integrated the [custom-audit](https://github.com/GoogleChrome/lighthouse/blob/f8a16b7eadf7e4d66b66dd656c4b058fa908503b/docs/recipes/custom-gatherer-puppeteer/custom-audit.js#L12) example in my existing user flow script .In the lighthouse html reports i am not able to see the results for custom-audit.

Does LH supports user flow integration with custom-audit?

**Comment by adamraine** [comment]
Yes, I can get it working with the following script (exact file paths will vary for your setup:

```js
import puppeteer from 'puppeteer';
import fs from 'fs';
import open from 'open';
import {startFlow} from 'lighthouse';

const browser = await puppeteer.launch({
  headless: false,
  ignoreDefaultArgs: ['--enable-automation'],
});
const page = await browser.newPage();

const {default: config} = await import('/Users/asraine/lighthouse/docs/recipes/custom-gatherer-puppeteer/custom-config.js');
const flow = await startFlow(page, {config, flags: {configPath: '/Users/asraine/lighthouse/docs/recipes/custom-gatherer-puppeteer/custom-config.js'}});

await flow.navigate('https://example.com');

const report = await flow.generateReport();
fs.writeFileSync('flow.report.html', report);
open('flow.report.html');

await browser.close();
```

**Reply by iamsethi** [reply]
@adamraine I've one follow up questions

1. Is it possible to see the custom audit reports in the summary tab. Currently i am able to see it after clicking on sidebar separator tab only

<img width="1533" alt="image" src="https://github.com/GoogleChrome/lighthouse/assets/12969857/079ca918-9195-491a-a62e-451d0cd07b34">

**Reply by adamraine** [reply, marked as answer]
Unfortunately that's not currently possible, but that's a good improvement idea for https://github.com/GoogleChrome/lighthouse/issues/15254
